builds/codex-v101x-aeturnum-prime/scripts/policy_eval.py

```
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
```

Ruff (0.14.3)

Shebang is present but file is not executable

(EXE001)